apps/api/src/app/shared/shared.module.ts

```diff
@@ -100,9 +100,8 @@ const PROVIDERS = [
         version: packageJson.version,
       })
     ),
-    QueuesModule,
   ],
   providers: [...PROVIDERS],
-  exports: [...PROVIDERS, LoggerModule, QueuesModule],
```

Comment:
Not sure why we need this one here therefore I removed it.
If someone knows the reason please let me know.

Reply:
It was added to provide the QueuesModule as provider in any module that is being dependant of SharedModule so in case of need QueuesModule would be available without having to inject it in said module.